joint-strategy-subcommittee/2018-02-21.md

```diff
@@ -0,0 +1,109 @@
+
+#### Present:
+
+- Mark Hinkle, Executive Director, Node.js Board 
+- Sarah Novotny, Board Member, Google
+- Todd Moore, Board Member, Node.js Chairman, IBM
+- Ashley Williams, Individual Director
+- William Kapke, Individual Director, 
+- Tierney Cyren, CommComm Co-Chair
+- Michael Dawson, TSC Chair
+- Myles Borin, Board Member, TSC
+- Tracy Hinds, Subcommittee Chair
+- Marten Mickos, Board Member, HackerOne
+- Brian McCallister, Board Member, Groupon  
+
+* * *
+##### Antitrust Compliance Notice
+
+Node.js Foundation meetings involve participation by industry competitors, and it is the intention of the Node.js Foundation to conduct all of its activities in accordance with applicable antitrust and competition laws. It is therefore extremely important that attendees adhere to meeting agendas, and be aware of, and not participate in, any activities that are prohibited under applicable US state, federal or foreign antitrust and competition laws. 
+
+Examples of types of actions that are prohibited at Node.js Foundation meetings and in connection with Node.js Foundation activities are described in the Node.js Foundation Antitrust Policy. If you have questions about these matters, please contact your company counsel, or if you are a member of the Node.js Foundation, feel free to contact Andrew Updegrove of the firm of Gesmer Updegrove LLP, which provides legal counsel to the Node.js Foundation.
+* * *
+  
+## AGENDA
+
+### Establish positions on...Expectations (20 minutes)
+- Project expectations of Board 
+- Project expectations for Executive Director 
+- Expectations of Foundation + OSS Project relationship 
+  
+
+### Expectations (20 minutes)
+- The Future of Individual Membership Program(20 minutes) 
+  
+## NOTES
+
+- Reviewed the agenda and purpose for today. 
+- Points of discussion for project expectations of the Board
+  - What support is expected? What responsibilities are expected in the relationship? 
+   - Clear definition of what the Board does 
+   - Manages money 
+   - Manages corporate membership 
+   - More of a relationship with the community, interact more 
+    - Initiatives from the project side need money. With cadence of board meetings, initiatives
+    can be left by the wayside. Ex: group license for IRCCloud for collaborators is an easy win
+    to talk about at the subcommittee level. 
+   - Be able to make requests to the subcommittee to help move proposals/requests forward 
+ 
+- What the Board does:
+  - Running the business, legal and marketing aspects of the Node.js Foundation 
+  - Prioritizing what investments to allocate funds to, and oversee a budget (e.g. fund build and test infrastructure) 
+  - Define the structure, process and terms for a Certification & Compliance program 
+  - Oversee Marketing such as PR/AR, branding via a members only marketing committee 
+  
+How do these not align?
+
+Early on in the founding, the Board spent a lot of energy on sharing information and managing expectations.
```

Comment:
more context on this point might help, i dont remember what it means

Reply:
ah, good point. Was in re: to the mystery of what the Board does